submissions/denismaysov/html-css-popup/index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="ru">
```

Comment:
You set russian language as the main page language but you use russian only as `alt` attribute value. Also, icons are decorative images, and decorative images must have empty `alt` value.